modules/UI/authentication/AuthHandler.js

```diff
@@ -111,14 +117,14 @@ function initJWTTokenListener(room) {
                 // the session-ID from the focus, which will then be used
                 // to upgrade current connection's user role
 
-                newRoom.room.moderator.authenticate()
```

Comment:
I don't think this will work, this function is in lib-jitsi-meet, and you didn't modify that.

Reply:
That is correct. Reverted.

Reply:
If you're referring to "authenticate", yes, I didn't modify that.  
I reverted "moderator.authenticateExternal" to "moderator.authenticate" to use the lib-jitsi-meet one 
and not the AuthHandler one.